src/main/flight/mixer.c

```diff
@@ -661,6 +661,130 @@ void StopPwmAllMotors()
 }
 
 #ifndef USE_QUAD_MIXER_ONLY
+static int16_t decidegreeToServoPulse(uint8_t servo_index, int16_t angle_deci_degree)
+{
+    // normalize angle to range -180 to 180 (degree)
+    while (angle_deci_degree > 1800)
+        angle_deci_degree -= 3600;
+    while (angle_deci_degree < -1800)
+        angle_deci_degree += 3600;
+
```

Comment:
yes, and it is the intended logic. Maybe i should set a bit of hysteresis to prevent it form "dancing" if on the limit?